src/Components/Web.Extensions/src/InputFile/IFileListEntry.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Components.Web.Extensions
+{
+    /// <summary>
+    /// Represents the data of a file selected from an <see cref="InputFile"/> component.
+    /// </summary>
+    public interface IFileListEntry
+    {
+        /// <summary>
+        /// Gets the name of the file.
+        /// </summary>
+        string? Name { get; }
+
+        /// <summary>
+        /// Gets the last modified date.
```

Comment:
For each of the nullable properties here, would it be possible to:

 * Verify if browsers might actually give null values in some cases
 * If so, update the XML docs so it clarifies this, e.g., `Gets the last modified data if supplied by the browser, otherwise null.`

This applies especially for reference-type things like `string`, because if a developer sees that (for example) `Type` is nonnull when they test it, they will assume we always guarantee it's not null.

Reply:
According to the FileAPI spec, it looks like browsers are required to at least provide a default value for every file attribute. As such, I believe `RelativePath` should be the only nullable property, since it's the only non-standard one.

Speaking of which, do you think `RelativePath` is something that we should even support at all? [This](https://developer.mozilla.org/en-US/docs/Web/API/File/webkitRelativePath) suggests that it probably won't ever become standardized, so I'm not sure if we want to risk getting stuck supporting a deprecated feature.

Reply:
> Speaking of which, do you think RelativePath is something that we should even support at all? This suggests that it probably won't ever become standardized, so I'm not sure if we want to risk getting stuck supporting a deprecated feature.

Great point. I'm guessing from the name it's already not supported on Firefox today.

Given the likely loss of this in the future, and probably non-universal support today, I agree it would be better to drop this property entirely.